inexor/engine/rendergl.cpp

```diff
@@ -581,6 +581,7 @@ void cef_resize(int width, int height)
         // TODO: not fully working
         spdlog::get("global")->info("Update Inexor User Interface Screen Size: {0}x{1}", width, height);
         cef_app->GetHudLayer()->Resize(0, 0, width, screen_manager.screenh);
+        cef_app->GetConsoleLayer()->Resize(0, 0, width, screen_manager.screenh);
```

Comment:
I think this codepart should get fixed with this PR: its not `screen_manager.screenh` but `height`. otherwise weird errors could occur someday